src/main/resources/com/puppycrawl/tools/checkstyle/checks/coding/messages.properties

```diff
@@ -40,6 +40,7 @@ multiple.variable.declarations.comma=Each variable declaration must be in its ow
 nested.for.depth=Nested for depth is {0,number,integer} (max allowed is {1,number,integer}).
 nested.if.depth=Nested if-else depth is {0,number,integer} (max allowed is {1,number,integer}).
 nested.try.depth=Nested try depth is {0,number,integer} (max allowed is {1,number,integer}).
+no.array.trailing.comma=Array should not contain trailing comma.
```

Comment:
plase use google translator to provide transalations to other languages

Reply:
This one too.